lms/templates/verify_student/intro_step.underscore

```diff
@@ -39,9 +39,9 @@
           <% if ( requirements['photo-id-required'] ) { %>
           <li class="req req-1 req-id">
             <h4 class="title"><%- gettext( "Photo ID" ) %></h4>
-            <div class="placeholder-art">
-              <i class="icon-list-alt icon-under"></i>
-              <i class="icon-user icon-over"></i>
+            <div class="placeholder-art fa-stack fa-lg">
+              <i class="icon fa fa-list-alt fa-stack-2x"></i>
+              <i class="icon fa fa-user fa-stack-1x"></i>
```

Comment:
Should be 
![screen shot 2015-01-09 at 10 02 48 am](https://cloud.githubusercontent.com/assets/2052185/5681466/c77a33e4-97e6-11e4-88c2-b8e0cafc8d94.png)